.dockerignore

```diff
@@ -1,5 +1,16 @@
 *
-!gitea
```

Comment:
After re-thinking if we add all the repo (remove .dockerignore) we could be able to ask to compile a specific version (tag) at build stage via git.